features/call/impl/src/main/kotlin/io/element/android/features/call/impl/utils/DefaultCallWidgetProvider.kt

```diff
@@ -48,7 +48,6 @@ class DefaultCallWidgetProvider @Inject constructor(
         ).getOrThrow()
 
         val driver = room.getWidgetDriver(widgetSettings).getOrThrow()
-        room.destroy()
```

Comment:
We cannot destroy the room here since it will close the FFI Room and the WidgetDriver needs it to work properly.

Reply:
Thanks, and sorry for the regression 😞 .